channeldb/meta_test.go

```diff
@@ -431,3 +431,35 @@ func TestMigrationReversion(t *testing.T) {
 			"want: %v, got: %v", ErrDBReversion, err)
 	}
 }
+
+// TestMigrationDryRun ensures that opening the database in dry run migration
```

Comment:
should also check that we receive `ErrMigrationOK`

Reply:
fixed